index.html

```diff
@@ -3,13 +3,15 @@
   <head>
     <meta charset="utf-8">
     <title>Startrly</title>
+    <link rel=stylesheet href="styles/normalize.css" type="text/css" media=screen>
+    <link rel=StyleSheet href="styles/style.css" type="text/css" media=screen>
   </head>
   <body>
-    <div>
+    <div class="container">
 
 
-      <nav>
-        <ul>
+      <nav class="nav-bar">
```

Comment:
I like the addition of these classes.  I think you did a great job of choosing descriptive names.